src/etc/inc/interfaces.inc

```diff
@@ -2578,8 +2578,11 @@ function interface_wireless_configure($if, &$wl, &$wlcfg) {
 
 	$wlcmd = array();
 	$wl_sysctl = array();
-	/* Make sure it's up */
-	$wlcmd[] = "up";
+	/* Disable this for now as we will bring up the interface as a last step.
+	 * Bringing up the interface with all the other arguments causes issues when using VAPs.
+	 * Related to #5616.
+	 */
+	//$wlcmd[] = "up";
```

Comment:
It's better to just remove the code here instead of commenting it out. We have history to see why it was removed